app/views/reference/index.njk

```diff
@@ -6,7 +6,7 @@
 
 {% block main_content %}
   <h1 class="heading-large page-title">
-    Pay for {{ productName }}
+    {{ productName }}
```

Comment:
I assume this string change has been agreed

Reply:
Removal of `Pay for` has been agreed with @soniaturcotte yesterday and there are other changes that will follow. Your assumption is correct 👍